modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/AnalyticsCloud/index.js

```diff
@@ -114,7 +114,7 @@ const ActivationStatusAnalyticsCloud = ({
 			title: 'Analytics Cloud Activation',
 		},
 		[STATUS_TAG_TYPE_NAMES.inProgress]: {
-			dropdownIcon: userAccount.isStaff && (
+			dropdownIcon: userAccount.isStaff && userAccount.isProvisioning && (
```

Comment:
I guess the stories need to be staff or provisioning user role.

```suggestion
			dropdownIcon: (userAccount.isStaff || userAccount.isProvisioning) && (
```